README.md

```diff
@@ -1347,6 +1352,7 @@ Manages a Cisco Network Interface. Any resource dependency should be run before
 | `ipv4_pim_sparse_mode` | Not supported on IOS XR |
 | `negotiate_auto` | Not supported on IOS XR |
 | `speed` | Not supported on IOS XR |
+| `stp_bpdufilter` <br> `stp_bpduguard` <br> `stp_cost` <br> `stp_guard` <br> `stp_link_type` <br> `stp_port_priority` <br> `stp_port_type` <br> `stp_mst_cost` <br> `stp_mst_port_priority` <br> `stp_vlan_cost` <br> `stp_vlan_port_priority` | Not supported on IOS XR |
```

Comment:
*nit* To be consistent with all other properties in this table, lets keep it to one property per row.